### DISCUSSION TITLE
Low gameplay performance

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#24050] [Q&A]
**Describe the bug:** when hitobject should be clicked (holded) game slows down (music not)
this happens mainly on windowed and borderless but it happens on fullscreen too, but it isn't thatb big then (but still sometimes noticable in gameplay)

**Screenshots or videos showing encountered issue:** https://streamable.com/9hbx7

**osu!lazer version:** 2019.830.0

**Logs:** 
[database.log](https://github.com/ppy/osu/files/3561369/database.log)
[network.log](https://github.com/ppy/osu/files/3561373/network.log)

[performance.log](https://github.com/ppy/osu/files/3561371/performance.log)
[runtime.log](https://github.com/ppy/osu/files/3561372/runtime.log)

**Comment by peppy** [comment]
Your game is running at 24fps. Please provide full PC specs and also let us know if it runs just as bad at other (non-gameplay) screens.

**Comment by ticotaco72** [comment]
Windowed:
main menu: 30-50 fps
song select: 5-20fps
gameplay: 40fps but not smoout hitobjects like described

borderless:
main menu: 40 fps
song select: 7-20fps
gameplay 30-40 fps with described issue

fullscreen: 
main menu:40-60fps
song select:10-15fps
gameplay: 40-70fps without the described issue

spec:
amd fx 4100 quadcore 3.6ghz x64
12 gb ram
windows 10 1903
ati radeon 3000 graphics

**Comment by peppy** [comment]
@ticotaco72 how are things looking for you these days performance-wise?

**Comment by Joppe27** [comment]
Are you all looking for more info on this? It looks like the original poster is not active anymore. I have severe performance issues too, so it would be nice if this got fixed eventually! I have a pretty similar hardware configuration and similar (worse) performance issues, on 2021.320.0 still, so please ask away if I can help out here.
Hardware: Intel Core 2 Duo P8600, ATI radeon HD3400, Windows 7

**Comment by peppy** [comment]
We continue to make performance improvements basically every release. Can't offer you more than that right now, so please keep checking back.

I'm only leaving this thread open to try and get a comparative look at performance from a user who was having issues several years ago, because things have already improved over the period of time. Knowing that you have issues at the current day is helpful, but not in the context of this thread.

**Comment by Joehuu** [comment]
Converting to a discussion as no action can be done here (this was left opened to track the OPs performance improvements over time).
